# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Action definition for choosing centroids
# The goal specifies the client id and the matrix with the lists in which each element is of the form [Reward, X, Y]
# The action server should try to choose the centroid that maximizes the information gain depending on the priority
# of the robots (r0 > ... > rn)

# Goal definition
string client_id
InfoMatrix matrix
int8 rows
